### DISCUSSION TITLE
index.html is the only path working in pb_public

### DISCUSSION THREAD

**Discussion by justmars** [pocketbase/pocketbase#2948] [Q&A] [answered]
Stared playing around with the API and really enjoy the dx involved. Wanted to see if I could create a full static site in `/pb_public` but it seems like only `/pb_public/index.html` works. Assuming I have:

1. `<a href='/about'>` inside index;
2. another file `pb_public/about.html`
3. clicking on the `<a>` will change the browser location to `/about`, make a network call to `/about` but the response still retains the original contents of `index.html`.

Am I correct that it's only if I register new routes via Pocketbase as a framework that non-index pages become "routable"? Phrased another way, is a multi-page application supported within `/pb_public` running Pocketbase as a single binary?

**Comment by ganigeorgiev** [comment, marked as answer]
This is expected. We currently resolve automatically only `index.html`. Any other file needs to be accessed via its full filepath, aka. in your case `/about.html`.

To access the html page without specifying the extension, you can create a `pb_public/about` directory and move your page as `index.html` inside it.

If you use a frontend framework with prerendering, usually there should be a config option that will output the generated static html files in their own directory. For example, SvelteKit has the `trailingSlash = 'always'` option - https://kit.svelte.dev/docs/page-options#trailingslash.

**Reply by justmars** [reply]
Thank you for the very prompt reply.

I'm toying with a simple site to explore the possibilities with just bare html and jinja. The creation of directories with index.html inside of it seems going outside the aim to simplify. Think I'll try to create a filter to output links using their fullpaths. Thank you again for such a quick and detailed response.
